.github/workflows/sanity-test.yaml

```diff
@@ -318,6 +321,53 @@ jobs:
           restore-args: '--folder ${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }} --restore-permissions'
           test-folder: '${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }}'
           log-dir: ${{ env.CORSO_LOG_DIR }}
+          with-export: true
+
+##########################################################################################################################################
+
+# Groups and Teams
+
+      # generate new entries for test
+      - name: Groups - Create new data
+        if: never() # TODO: enable when ready
```

Comment:
I don't think `never()` is available. I think we can however just use `false` though.

Reply:
so it seems.  Good catch.